src/components/styles/PageContainer.styled.js

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components';
+
+export const PageContainer = styled.div`
+  background-color: ${({ backgroundColor }) => backgroundColor};
+  height: auto;
+`;
```

Comment:
is it possible for this component to instead of getting the background color to change it directly here?